code/controllers/subsystem/discord.dm

```diff
@@ -73,6 +73,27 @@ SUBSYSTEM_DEF(discord)
 	for(var/url in config.discord_admin_webhook_urls)
 		SShttp.create_async_request(RUSTG_HTTP_METHOD_POST, url, dwp.serialize2json(), list("content-type" = "application/json"))
 
+/datum/controller/subsystem/discord/proc/send2discord_simple_nomentors(content)
```

Comment:
Make this have `check_send_always` as a parameter too, or rename it, since this is gonna send always, despite the name being `nomentors`.

Reply:
These could do with some documentation really, the `check_send_always` argument in particular is pretty ambiguous.
It could do any of these and still be valid:
1. Check if the message should always be sent
2. Check if 'always' should be sent
3. Always send the check
4. Always check 'send'
5. etc

I'll probably just rename the proc since there's no cases when mentorhelps *shouldn't* be sent to the discord channel.